src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -5349,5 +5349,39 @@ static void Main()
             string expectedOutput = @"1";
             CompileAndVerify(source, expectedOutput: expectedOutput);
         }
+
+
+        [Fact]
```

Comment:
>[Fact] [](start = 8, length = 6)

It looks like there ate two empty lines before this method, consider removing one.

Reply:
Done.